src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +54,46 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        if (dateParams.length != 3) {
+            return Optional.empty();
+        }
+        try {
+            return Optional.of(LocalDate.of(dateParams[0], dateParams[1], dateParams[2]));
+        } catch (Exception ex) {
+            return  Optional.empty();
+        }
     }
 
     /**
      * Дано время и на сколько часов нужно его изменить.
      * Верните измененное время на указаную величину.
      */
     public LocalTime addHours(LocalTime localTime, Integer hoursToAdd) {
-        return LocalTime.now();
+        return localTime.plusHours(Long.valueOf(hoursToAdd));
```

Comment:
```suggestion
        return localTime.plusHours(hoursToAdd);
```
the same in other methods